package.json

```diff
@@ -16,6 +16,7 @@
     "@types/react-dom": "^18.0.6",
     "algoliasearch": "^4.14.2",
     "instantsearch.js": "^4.47.0",
+    "netlify-cli": "^12.7.2",
```

Comment:
Move to devDependencies -> validate

Reply:
Since this is an app (not a library), I don't see many reasons to split dependencies. Additionally, there are different views on how to split them for applications. I would leave it together for now as there is nothing wrong with it and get back to this in the future should we encounter some convincing arguments to split the dependencies.